test/render-poll.js

```diff
@@ -0,0 +1,19 @@
+export function renderPoll(poll) {
```

Comment:
Everything is working, but you don't want this render-poll file to live in your test folder. The test folder should only be for test-related files.